### ISSUE TITLE
System Hang

### ISSUE THREAD

**Issue by Lurkios** [lltcggie/waifu2x-caffe#83]
Unkillable process waifu2x-caffe.exe, the process hangs. The window never appears, but the process does show up in task manager. The process cannot be terminated even with /f /t (force and kill child process) switches. System shutdown or reboot fails (including with /force).

waifu2x-caffe.exe opens the command window, but then displays the same behavior as above when attempting to exit the window.

Only way out seems to be a hard reset.

**Comment by YukiSakuma** [comment]
Do you have Microsoft Security Essentials installed and running? Try killing the process msmpeng.exe if you have it and running using procmon.

**Comment by Lurkios** [comment]
@YukiSakuma No, I don't. I'm using Kaspersky - and that process isn't running. Unfortunately it's kind of hard to test, since I can't replicate the issue in a VM and doing it on my desktop system risks desyncing my RAID (again), which is a 2-day rebuild.

**Comment by YukiSakuma** [comment]
Open up process explorer and waifu2x-caffe.exe and then ctrl+f  search waifu2x-caffe.exe, see if there's any process trying to "hang" it.

**Comment by Lurkios** [comment]
@YukiSakuma Nothing at all. Nothing hanging it, no child process. Just it.

**Comment by YukiSakuma** [comment]
Did you try older versions of w2x caffe?

**Comment by Lurkios** [comment]
@YukiSakuma No, I haven't. I was hoping it was a known issue with some potential workaround. Unfortunately I can't really test either, with the above RAID issue.

**Comment by rombat** [comment]
I have the exact same issue. And what's weird is that I tried waifu2x-caffe for the first time this morning (fresh CUDA install) and it was working just fine. And after a reboot, it's not working anymore. The process is launched but I can't see anything and can't kill it.

**Comment by nagadomi** [comment]
I guess it's a GPU driver problem. The installed driver will be applied at reboot.
Installing the latest drivers that is compatible with your GPU may solve it.
